modules/installation-configuration-parameters.adoc

```diff
@@ -407,6 +407,16 @@ endif::ibm-power-vs[]
 Set the `networking.machineNetwork` to match the CIDR that the preferred NIC resides in.
 ====
 
+|networking:
+  ovnKubernetesConfig:
+    ipv4:
+      internalJoinSubnet:
+|
+Configures the IPv4 subnet that is used internally by `ovn-kubernetes`.
```

Comment:
ovn-kubernetes uses more than one subnet, so I would either say "configures a subnet" or, better:

Configures the IPv4 join subnet used internally...